mycroft/audio/service.py

```diff
@@ -202,6 +202,15 @@ def handle_stop(self, message):
             self.tts.playback.clear()  # Clear here to get instant stop
             self.bus.emit(Message("mycroft.stop.handled", {"by": "TTS"}))
 
+    def handle_queue_audio(self, message):
+        viseme = message.data.get("viseme")
+        ident = message.data.get("ident") or message.context.get("ident")  # unused ?
+        audio_ext = message.data.get("audio_ext")  # unused ?
+        audio_file = message.data.get("filename")
```

Comment:
Should we return or raise an exception if this is None? Otherwise we'll throw an exception in the next line

Reply:
we can do that, but this should never be missing as that is the whole point of this method....

a proper LOG can't hurt

Reply:
Yeah; mostly wanted to make sure it's obvious what failed if you include something like `file` instead of `filename` in the message data. Maybe we should check for file existence too? Could help troubleshoot relative path errors, `~/`-relative paths, etc